src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -56,16 +55,16 @@ protected override SyntaxNode GetObjectCreationNewExpression(SyntaxTree tree, in
             return CSharpSyntaxContext.CreateContext(document.Project.Solution.Workspace, semanticModel, position, cancellationToken);
         }
 
-        protected override async Task<ImmutableArray<ISymbol>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<(ISymbol symbol, CompletionItemRules rules)>> GetPreselectedSymbolsWorker(SyntaxContext context, int position, OptionSet options, CancellationToken cancellationToken)
         {
             var result = await base.GetPreselectedSymbolsWorker(context, position, options, cancellationToken).ConfigureAwait(false);
             if (result.Any())
             {
-                var type = (ITypeSymbol)result.Single();
+                var type = (ITypeSymbol)result.Single().symbol;
                 var alias = await type.FindApplicableAlias(position, context.SemanticModel, cancellationToken).ConfigureAwait(false);
                 if (alias != null)
                 {
-                    return ImmutableArray.Create(alias);
+                    return ImmutableArray.Create((alias, CompletionItemRules.Default));
```

Comment:
@Pilchie as I told @CyrusNajmabadi this is just placeholders for now, I'm just trying to do one thing at a time, so my approach is to let the tuple flow through the system and later on deal with the rules that are needed and also improve the quality of the code when it comes to allocations or whatever.